alter_configs_request.go

```diff
@@ -0,0 +1,146 @@
+package sarama
+
+type AlterConfigsRequest struct {
+	Resources    []*AlterConfigsResource
+	ValidateOnly bool
+}
+
+type AlterConfigsResource struct {
+	T             ResourceType
+	Name          string
+	ConfigEntries []*ConfigEntryKV
+}
+
+type ConfigEntryKV struct {
+	Name  string
+	Value string
```

Comment:
values are supposed to be nullable, this should probably be a `*string` or something?

Reply:
Whoops! you're right, I'll update